src/main/java/uk/ac/ebi/ena/readtools/fastq/ena/Fastq2Sam.java

```diff
@@ -33,6 +33,8 @@
 
 public class Fastq2Sam {
 
+    private long totalRecordCount = 0, totalBaseCount = 0;

```

Comment:
Would it be possible please to replace 'Record' with 'Read' in totalRecordCount and in other places?